### DISCUSSION TITLE
Propper versions style guide

### DISCUSSION THREAD

**Discussion by AndrewStoyan** [superRaytin/react-monaco-editor#838] [Ideas]
It will be much better if repo will change major version if there is breakable changes. For example: 0.39.x requires React v16.x and 0.40.x already Recat v17.x.
If repo will change major version it will be compatible with npm behavior.

**Comment by domoritz** [comment]
According to semantic versioning, breaking changes are allowed in 0.x to 0.x+1. Yes, it would make sense to jump to 1.0 at some point to make minor version changes easier.
